controllers/set/status.go

```diff
@@ -8,34 +8,44 @@ import (
 	"github.com/metal-stack/metal-lib/pkg/pointer"
 )
 
+type FirewallConditionStatus struct {
```

Comment:
I think the structs and the fields do not need to be public. It's just an internal helper?